crates/release_plz/src/config.rs

```diff
@@ -264,6 +264,10 @@ pub struct PackageConfig {
     /// Whether to create/update changelog or not.
     /// If unspecified, the changelog is updated.
     pub changelog_update: Option<bool>,
+    /// # Features Always Increment Minor Version Flag
```

Comment:
```suggestion
    /// # Features Always Increment Minor Version
```
the "flag" suffix is not present anywhere else.